crypto/src/signature/mod.rs

```diff
@@ -633,4 +658,27 @@ mod tests {
         }
         // From the above we can conclude that `SignatureWrapperOf` have consistent behavior for `HashSet` and `BTreeSet`
     }
+
+    #[test]
+    fn signature_serialized_representation() {
+        let input = json!({
+            "public_key": "e701210312273E8810581E58948D3FB8F9E8AD53AAA21492EBB8703915BBB565A21B7FCC",
```

Comment:
The fact that pubkey is serialized as uppercase (mostly?), but payload is lowercase is not very nice..

Can we make it consistent?

Reply:
This goes back to https://github.com/hyperledger/iroha/pull/3316. TBH, I don't understand how this

> improves private key validation

@mversic, could you comment on that?

Reply:
> This goes back to https://github.com/hyperledger/iroha/pull/3316. TBH, I don't understand how this
> 
> > improves private key validation
> 
> @mversic, could you comment on that?

This was referring to something else. Uppercase/lowercase has no influence on validation

I would prefer making it all uppercase

Reply:
Changed `PrivateKey`'s and `Signature`'s `payload` serialized hex to uppercase.

I haven't bothered to updated every single hex in the workspace to uppercase accordingly. Hope it's fine.